xtra-bitmex-price-feed/src/lib.rs

```diff
@@ -131,10 +135,16 @@ pub enum Error {
 #[derive(Debug)]
 struct NewQuoteReceived(Quote);
 
-/// Request the latest quote from the price feed.
+/// Request the latest quote for a given contract symbol from the price feed.
 #[derive(Debug, Clone, Copy)]
 pub struct LatestQuote(pub ContractSymbol);
```

Comment:
Is this still being used?

Reply:
nope, after a while of having them both I realised it was not needed...